test/acceptance/framework/config/config.go

```diff
@@ -2,23 +2,23 @@ package config
 
 // TestConfig holds configuration for the test suite.
 type TestConfig struct {
-	NoCleanupOnFailure bool
-	ECSClusterARN      string `json:"ecs_cluster_arn"`
-	Subnets            interface{}
-	Suffix             string
-	Region             string
-	VpcID              string `json:"vpc_id"`
-	LogGroupName       string `json:"log_group_name"`
-	Tags               interface{}
+	NoCleanupOnFailure    bool
+	ECSClusterARN_Fargate string `json:"ecs_cluster_arn_fargate"`
+	ECSClusterARN_EC2     string `json:"ecs_cluster_arn_ec2"`
```

Comment:
what if we rename them a bit to remove underscores and also keep readable, e.g `FargateClusterARN` and `EC2ClusterARN`?